src/Tizen.NUI/src/public/Common/Matrix.cs

```diff
@@ -77,6 +75,35 @@ public static Matrix IDENTITY
             }
         }
 
+        public float this[uint index]
+        {
+            set
+            {
+                SetValueAtIndex(index, value);
+            }
+            get
+            {
+                return ValueOfIndex(index);
+            }
+        }
+
+        public static Vector4 operator *(Matrix arg1, Vector4 arg2)
```

Comment:
Does this become public method?
Then arg1 and arg2 are not so fancy for parameter name of public method.